Update brother.markdown

miklosbagi opened the pull request:
Added v1/v2c read-only SNMP requirement to increase community success rate with this integration.

![Screenshot 2021-01-05 at 17 16 57](https://user-images.githubusercontent.com/68903245/103670460-e239ad80-4f79-11eb-84f0-dd52a2ae7971.png)

-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - | made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

miklosbagi committed:
Update brother.markdown

Added v1/v2c read-only SNMP requirement as per my experience.

frenck commented on source/_integrations/brother.markdown:
```diff
@@ -26,3 +26,9 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
+
+## Configuring the printer
+
+SNMP settings are commonly under Network / Protocol / SNMP.
```
I think it might help if we mention that this is on the interface of the device somewhere?
The reason for this, is that this is the Home Assistant documentation and one might start looking for this in the Home Assistant configuration panels.

miklosbagi replied:
Yeah that makes sense, let me add that.
Is it OK to add the image I used in the pullreq to better illustrate where this setting is?

miklosbagi committed:
Update brother.markdown

Incorporating changes requested by @frenck, added img on how the config interface looks like (on latest brother firmware on 2340dw)

frenck commented on source/_integrations/brother.markdown:
```diff
@@ -26,3 +26,11 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
+To set SNMP, navigate to the printer's web interface (for example: http://192.168.5.6) and turn on under Network / Protocol / SNMP.
+For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access` is the option required (under advanced settings).
+
+![SNMP settings on Brother Printer web interface](https://user-images.githubusercontent.com/68903245/103670460-e239ad80-4f79-11eb-84f0-dd52a2ae7971.png)
```
Images need to be stored and hosted from our own website. If you add graphics, please add it to this repository.

miklosbagi replied:
oh ok - rolling...

Commit:
added webui image

frenck commented on source/_integrations/brother.markdown:
```diff
@@ -26,3 +26,10 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
+To set SNMP, navigate to the printer's web interface (for example: http://192.168.5.6) and turn on under Network / Protocol / SNMP.
+For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access` is the option required (under advanced settings).
+
+![SNMP settings on Brother Printer web interface](/source/images/integrations/brother/brother-printer-webui.png)
```
```suggestion
![SNMP settings on Brother Printer web interface](/images/integrations/brother/brother-printer-webui.png)
```

frenck committed:
:ambulance: Fix image link

frenck commented on source/_integrations/brother.markdown:
```diff
@@ -26,3 +26,10 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
+
+## Configuring the printer
+
+To set SNMP, navigate to the printer's web interface (for example: http://192.168.5.6) and turn on under Network / Protocol / SNMP.
```
Small tweak to prevent a dead link.

```suggestion
To set SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP.
```

frenck committed:
:pencil2: Tweak

frenck reviewed (approved):
Thanks, @miklosbagi 👍 

Happy New Year 🥂
